index.html

```diff
@@ -145,12 +145,31 @@
 	.container {
 	  margin-top: 0;
 	}
+  }
 
-	.hero {
-	  margin-top: 0;
-	  margin-bottom: 24px;
+@media (prefers-color-scheme: light) {
+	body {
+		background-color: white;
+		background: url('/assets/cloud_bg.svg');
+		background-size: cover;
+		background-position: center;
+		background-attachment: fixed;
+		line-height: 1.6;
 	}
-  }
+}	
+
+@media (prefers-color-scheme: dark) {
+	body {
+		background: black url('/assets/dark_bg.svg');
```

Comment:
Only add items that change in the @media block, the rest will use the same.values you set in the default.